mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -126,7 +105,7 @@ void ControllerAction::run(GoalHandle &goal_handle, AbstractControllerExecution
   {
     fillExePathResult(geometry_msgs::PoseStamped(), geometry_msgs::PoseStamped(),
                       mbf_msgs::ExePathResult::INVALID_PATH, "Controller started with an empty plan!", result);
-    goal_handle.setAborted(result, result.message);
+    concurrency_slots_[slot].goal_handle.setAborted(result, result.message);
```

Comment:
See previous comment: We could use the benefit of references here. Same for the following lines with the map access.